HPCC-17317 DOCS: ECLWatch Add Delete Part

g-pan opened the pull request:
Fix HPCC-17317 DOCS:ECLWatch Add Delete Part
Add documentation to the ECL Watch doc on
using the Roxie Package Maps, Add, Delete, Activate Part.

New Pull request, Replacing Pull Request # 10030 - due to substantial changes and some technical difficulties.

@JamesDeFabia please review
@miguelvazq please review
@afishbeck please review

Build results and PDF can be reviewed at: 
http://10.240.32.243/view/Docs/job/DocDev0IV/CE=ubuntu-14.04-amd64-1/lastSuccessfulBuild/artifact/build-docs/Release/docs/The_ECL_Watch_Manual-6.3.0-trunk0.pdf#page=93
Note that chrome users may need to download the file for viewing.

<!-- Thank you for submitting a pull request to the HPCC project

 PLEASE READ the following before proceeding.

 This project only accepts pull requests related to open JIRA issues.
 If suggesting a new feature or change, please discuss it in a JIRA issue first.
 If fixing a bug, there should be an issue describing it with steps to reproduce.
 The title line of the pull request (and of each commit within it) should refer to the
 associated issue using the format:

 HPCC-nnnnn Short description of issue

 This will allow the Jira ticket to be automatically updated to refer to this pull request,
 and will ensure that the automatically-generated changelog is properly formatted.
 Where a pull request contains a single commit the pull request title will be set automatically,
 assuming that the commit has followed the proper guidelines.

 Please go over all the following points, and put an `x` in all the boxes that apply. You may find
 it easier to press the 'Create' button first then click on the checkboxes to edit the comment.
-->

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [x] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browers
  - [ ] The component(s) render as expected

## Testing:
<!-- Please describe how this change has been tested.-->

<!-- Thank you for taking the time to submit this pull request and to answer all of the above-->

g-pan committed:
HPCC-17317 DOCS:ECLWatch Add Delete Part

Fix HPCC-17317 DOCS:ECLWatch Add Delete Part
Add documentation to the ECL Watch doc on
using the Roxie Package Maps, Add, Delete, Activate Part

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

HPCCSmoketest commented:
Automated Smoketest: :x:
Sha: 0effa5d5fadd8b0fdf63ab00e794e8556cc38877
Build: failed
Number of Error(s): 85
/mnt/disk1/home/vamosax/smoketest/PR-10050/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10050/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10050/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10050/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10050/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10050/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10050/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3068: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10050/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3100: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10050/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3100: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10050/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3100: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10050/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3100: parser error : StartTag: invalid 
 ... ( comment is too long, 92 lines, 13541 bytes, truncated) ... 
 Building CXX object aws-cpp-sdk-core/CMakeFiles/aws-cpp-sdk-core.dir/source/client/CoreErrors.cpp.o
[100%] Building CXX object aws-cpp-sdk-sqs/CMakeFiles/aws-cpp-sdk-sqs.dir/source/SQSErrors.cpp.o
make: *** [all] Error 2
Get ECLWatch build result


```

richardkchapman commented:
Needs rebasing for a clean merge

g-pan committed:
HPCC-17317 DOCS:ECLWatch Add Delete Part

Fix HPCC-17317 DOCS:ECLWatch Add Delete Part
Add documentation to the ECL Watch doc on
using the Roxie Package Maps, Add, Delete, Activate Part

Signed-off-by: g-pan <greg.panagiotatos@lexisnexis.com>

HPCCSmoketest commented:
Automated Smoketest: :x:
Sha: 43d23290fd9709a61359978871075701c43c9569
Build: failed
Number of Error(s): 85
checking for memory.h... /mnt/disk1/home/vamosax/smoketest/PR-10050/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3048: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10050/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3048: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10050/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3048: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10050/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3048: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10050/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3048: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10050/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3048: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10050/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3048: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10050/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3078: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10050/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3078: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10050/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3078: parser error : StartTag: invalid element name
/mnt/disk1/home/vamosax/smoketest/PR-10050/HPCC-Platform/docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml:3078: parser e
 ... ( comment is too long, 92 lines, 13563 bytes, truncated) ... 
 Building CXX object aws-cpp-sdk-core/CMakeFiles/aws-cpp-sdk-core.dir/source/client/CoreErrors.cpp.o
[100%] Building CXX object aws-cpp-sdk-sqs/CMakeFiles/aws-cpp-sdk-sqs.dir/source/SQSErrors.cpp.o
make: *** [all] Error 2
Get ECLWatch build result


```

g-pan commented:
Can't seem to find the error. Job builds in all tests: http://10.240.32.243/job/DocDev0IV/CE=ubuntu-14.04-amd64-1/

Conflicts: Would seem that the LOCALH17317 branch should win. 
Tried to resolve through this new pull request, new commits, but still getting conflicts. 

Once content approved, will rebase for merge.

g-pan commented:
Overly complicated to debug a simple change. Closing in order to issue a clean one.
